### PR TITLE
github-actions: fix codecov for unittests

### PR DESCRIPTION
**Pull request by victorjulien**
Don't overwrite ut coverage with later tests.

### CODE REVIEW

File: .github/workflows/builds.yml [hunk 1]
```diff
@@ -1514,14 +1514,26 @@ jobs:
...
           LLVM_PROFILE_FILE: "/tmp/la.profraw"
-      - run: llvm-profdata-14 merge -o ut.profdata /tmp/la.profraw
+      - run: llvm-profdata-14 merge -o la.profdata /tmp/la.profraw
+      - run: ./src/suricata --dump-features -c suricata.yaml -l /tmp
+        env:
+          LLVM_PROFILE_FILE: "/tmp/dumpfeatures.profraw"
+      - run: llvm-profdata-14 merge -o dumpfeatures.profdata /tmp/dumpfeatures.profraw
+      - run: ./src/suricata --dump-config -c suricata.yaml -l /tmp
+        env:
+          LLVM_PROFILE_FILE: "/tmp/dumpconfig.profraw"
```
Comment (jasonish) [inline .github/workflows/builds.yml:1535]: It matches the existing pattern, but how does the dumpfeatures profile data end up being uploaded to codecov?

Comment (victorjulien) [reply .github/workflows/builds.yml:1535]: IIRC it scans for profdata files

Comment (jasonish) [reply .github/workflows/builds.yml:1535]: All I see it doing is uploaded the coverage.txt file, which I never see this one merged into. Anyways, not really a thing with this PR.

**Review by jasonish — APPROVED** [review]

**Comment by victorjulien** [comment]
Merged in #12185, thanks!

**Pull request closed without merging**